tests/OSPSuite.Infrastructure.Tests/Import/DataSetToDataRepositoryMapperSpecs.cs

```diff
@@ -12,14 +12,40 @@
 
 namespace OSPSuite.Infrastructure.Import
 {
+   public class DimensionForUnitTests : Dimension
```

Comment:
We have some dimensions available already for testing. DomainHelperForSpecs.xxx Please use those

Reply:
Done